hal/armv7m/imxrt/10xx/hal.c

```diff
@@ -96,8 +97,23 @@ addr_t hal_kernelGetAddress(addr_t addr)
 }
 
 
+int hal_memAddMap(addr_t start, addr_t end, u32 attr, u32 id)
+{
+	return mpu_mapAlloc(start, end, attr, id);
+}
+
```

Comment:
Body if `mpu_mapAlloc()` should be placed inside this function.

Reply:
Possible, but not the use case because `mpu.c` is `hal` part and transferring one function `mpu_mapAlloc` will transfer all of `mpu.c` implementation to` hal.c` for each imxrt target (this leads to doubling code).